### DISCUSSION TITLE
How to display the album art from cover.jpg?

### DISCUSSION THREAD

**Discussion by halloleo** [TortugaPower/BookPlayer#835] [Q&A] [answered]
I have an audiobook containing many chapter mp3s. In the same folder I have a 500x500px `cover.jpg` file with the album art.

When I zip that folder, import it into _BookPlayer_ and then unpack it in _BookPlayer_ as a 'Volume" the book is played perfectly - but the album art is not shown.

How can I tell _BookPlayer_ to show the album art?

**Comment by EmeraldSword8** [comment]
I’d really like to know the answer to this as well

**Comment by halloleo** [comment, marked as answer]
One thing which does work, is embedding the art work in the MP3 file. 

This might be not ideal however, if your file repository of audiobooks follows the art-in-the-cover.jpg sidecar file convention.

(Marked as answer, but the answer _content_ is in @GianniCarlo's thread reply below.)

**Reply by EmeraldSword8** [reply]
Is there a way to do this on iOS?

**Reply by halloleo** [reply]
Not sure whether there exist apps on iOS to embed album art in  MP3.

**Reply by GianniCarlo** [reply]
Hey guys, sorry I was out last week, right now the only way to properly show the album art, is for it to be embedded in the file before importing it into the app. We do have plans to enable an interface to update/modify the artwork for a given book/volume, but that will come in the next major release v5 where we are making space for the feature. 

Side note: if you're on a Mac, and mainly use MP3s, check out [Forecast](https://overcast.fm/forecast), it's free and it lets you add chapter markers and artwork to an mp3 file. Another alternative is to bind the mp3 files into a single m4b file with [Audiobook Binder](http://bluezbox.com/audiobookbinder.html) which I think has some limitations of 23h files (it will generate a second file if the book goes over that threshold), but it lets you encode the artwork as well

**Reply by halloleo** [reply]
Thanks @GianniCarlo. I've used Bookbinder and it's ok. Also [fre:ac](https://www.freac.org/) is suppose to do it, but now I might give Forecast a try. Looks like it is the most convenient tool...

**Reply by gandalftheking** [reply]
Please intro this in v5 asap - appreciated
